packages/synapse-react-client/src/components/QueryContext/QueryContext.tsx

```diff
@@ -123,42 +107,14 @@ export const QueryContextProvider = ({
  * @template ExcludeOtherFields - If set to true, will remove optional fields besides those specified in TIncludedFields,
  * default false
  */
-export function useQueryContext<
-  TIncludedFields extends OptionalQueryBundleRequestFields = never,
-  ExcludeOtherFields extends true | false = false,
->(): QueryContextType<TIncludedFields, ExcludeOtherFields> {
-  const context = useContext(QueryContext) as QueryContextType<
-    TIncludedFields,
-    ExcludeOtherFields
-  >
+export function useQueryContext(): QueryContextType {
+  const context = useContext(QueryContext)
   if (context === undefined) {
     throw new Error('useQueryContext must be used within a QueryWrapper')
   }
   return context
 }
 
-export function usePaginatedQueryContext(): PaginatedQueryContextType {
```

Comment:
Removed `Paginated`/`Infinite` context types. Pagination is now handled internally.